### PR TITLE
Support for TFA Drop 30.3233.01

### PR DESCRIPTION
**Pull request by mhaas**
Tests in https://github.com/merbanan/rtl_433_tests/pull/324

TODO:

- [x] Initial support
- [x] Checksum/parity bits
- [x] Document preamble
- [x] Document fixed byte 
- [x] Enable device by default

### CODE REVIEW

**Comment by mhaas** [comment]
Hi @zuckschwerdt, I think I am pretty much done here - let me know if I can do anything to improve the PR!

Have a nice weekend!

File: src/devices/tfa_drop_30.3233.01.c [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+*/
+
+#include "decoder.h"
+
+#define TFA_DROP_BITLEN 66
+#define TFA_DROP_STARTBYTE 0x3 /* Inverted already */
+#define TFA_DROP_MINREPEATS 2
+
+static uint8_t compute_checksum(uint8_t *msg, int bytes);
+
```
Comment (merbanan) [inline src/devices/tfa_drop_30.3233.01.c]: Just a tiny nit not needed for merge. Move the function here to skip the future declaration part. Or move the checksum calculation to the util file.

**Comment by merbanan** [comment]
Looks good to me.

**Comment by zuckschwerdt** [comment]
Please note that I just changed the model from `TFA-Drop-30.3233.01` to `TFA-Drop`.
Model is only a short key of "alphanum dash alphanum", not a full description.

Sorry, I didn't notice that on merge.

**Pull request merged**